newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -103,6 +104,11 @@ export default function SummaryPage() {
 
   const mailSent = sendMailResponse !== null;
 
+  const createEvent = async (event, {value}) => {
+    const resp = await _createEvent(newdle.title, newdle.duration, newdle.final_dt);
+    window.open(resp['url'], '_blank');
```

Comment:
- Isn't `_blank` the default?
- I think this may trigger popup blockers depending on how long the request takes. I'd just redirect tbh.
- `resp.url`

Reply:
It indeed is. Updated `resp['url']` to `resp.url`.